src/client/auth/scram.rs

```diff
@@ -142,12 +146,8 @@ impl ScramVersion {
         // Normal SCRAM implementations would cease here. The following round trip is MongoDB
         // implementation specific and just consists of a client no-op followed by a server no-op
         // with "done: true".
-        let noop = doc! {
-            "saslContinue": 1,
-            "conversationId": server_final.conversation_id().clone(),
-            "payload": Bson::Binary(Binary { subtype: BinarySubtype::Generic, bytes: Vec::new() })
-        };
-        let command = Command::new("saslContinue".into(), source.into(), noop);
+        let noop = SaslContinue::new(server_final.conversation_id().clone(), Vec::new());
```

Comment:
The changes to this module are just to use the abstracted `sasl` module functionality that was pulled out.